tests-legacy/resources/modules/cronjobs/cronjobs.php

```diff
@@ -693,7 +693,7 @@ protected function updateWebservice($use_webservice)
         );
 
         $context_options = array('http' => array(
-            'method' => (is_null($webservice_id) == true) ? 'POST' : 'PUT',
+            'method' => ((null === $webservice_id) == true) ? 'POST' : 'PUT',
```

Comment:
No need to modify these files as they are not run.

Reply:
I understand that the tests are not run, but can we merge all the changes or do you want me to revert some of the changes? :)

Reply:
I think it's ok for now: it was stupid before, it's still stupid after and this introduces no regression :D